.changeset/ripe-fans-push.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Updates `primsjs` to version 1.30.0, which fixes a security advisory and adds support for more languages.
```

Comment:
Is it worth mentioning that the vulnerability doesn't affect Astro users in general (if that's true), because it doesn't use untrusted inputs?

Reply:
Sure, feel free to provide a suggestion

Reply:
```suggestion
Updates `primsjs` to version 1.30.0, which adds support for more languages and fixes a security advisory which does not affect Astro.
```